docs/merkletree/treetex/main.go

```diff
@@ -169,54 +175,56 @@ func perfectMega(prefix string, height uint, leafIndex uint64) {
 }
 
 // perfect renders a perfect subtree.
-func perfect(prefix string, height uint, index uint64) {
-	perfectInner(prefix, height, index, true)
+func perfect(prefix string, height uint, index uint64, nodeText nodeTextFunc) {
+	perfectInner(prefix, height, index, true, nodeText)
 }
 
 // drawLeaf emits TeX code to render a leaf.
-func drawLeaf(prefix string, index uint64) {
+func drawLeaf(prefix string, index uint64, text string) {
 	a := nInfo[compact.NewNodeID(0, index)]
-	fmt.Printf("%s [%d, %s, tier=leaf]\n", prefix, index, a.String())
+	fmt.Printf("%s [%s, %s, tier=leaf]\n", prefix, text, a.String())
 }
 
 // openInnerNode renders TeX code to open an internal node.
 // The caller may emit any number of child nodes before calling the returned
 // func to close the node.
 // Returns a func to be called to close the node.
-func openInnerNode(prefix string, id compact.NodeID) func() {
+func openInnerNode(prefix string, id compact.NodeID, text string) func() {
 	attr := nInfo[id].String()
-	fmt.Printf("%s [%d.%d, %s, tier=%[2]d\n", prefix, id.Level, id.Index, attr)
+	fmt.Printf("%s [%s, %s, tier=%d\n", prefix, text, attr, id.Level)
```

Comment:
Does it look better without `id.Index`?

Reply:
That's built in (or not, depending on the formatting chosen) to the `text` label now